vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -179,6 +232,72 @@ function Test-FilesToCacheOnVHD
 
 }
 
+function Test-ValidateAllSignature {
+    foreach ($dir in $map.Keys) {
+        Test-ValidateSinglePackageSignature $dir
+    }
+}
+
+function Test-ValidateSinglePackageSignature {
+    param (
+        $dir
+    )
+
+    foreach ($URL in $map[$dir]) {
+        $fileName = [IO.Path]::GetFileName($URL)
+        $dest = [IO.Path]::Combine($dir, $fileName)
+
+        if(![System.IO.File]::Exists($dest)) {
+            Write-ErrorWithTimestamp "File $dest does not exist"
+            $invalidFiles = $invalidFiles + $dest
```

Comment:
I think that we do not need this since we have called Test-FilesToCacheOnVHD.

Reply:
Removed.